dashboard/test/lib/services/lti_test.rb

```diff
@@ -498,6 +498,55 @@ class Services::LtiTest < ActiveSupport::TestCase
     assert_equal lti_section.followers.last, user_to_remove
   end
 
+  test 'should update a user\'s name when syncing a section' do
```

Comment:
This is not something to change now, but by the length of this test, we may benefit by switching to specs style.